webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx

```diff
@@ -90,7 +85,7 @@ const SystemMetricChart: React.FC<ChartProps> = ({ getOptionsForMetrics, trial }
         )
       }
       title="System Metrics">
-      <UPlotChart data={systemMetrics.data} options={options} />
+      <LineChart series={systemMetrics.data} xAxis="Time" />
```

Comment:
Can this be `XAxisDomain.Time` ?

Reply:
According to the original chart [latest-master](http://latest-master.determined.ai:8080/det/experiments/1783/profiler?agentId=i-0c5355850449c0f1b&name=disk_throughput_write), x-axis is date time, so i used `Time`

Reply:
in this case I think you want `xLabel="Time" xAxis={XAxisDomain.Time}`, maybe it would work with just `xAxis={XAxisDomain.Time}`

Reply:
ah i got what you meant. ok